galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager/src/main/java/dev/galasa/http/internal/HttpManagerImpl.java

```diff
@@ -61,6 +68,28 @@ public void initialise(@NotNull IFramework framework, @NotNull List<IManager> al
             if (!ourFields.isEmpty()) {
                 youAreRequired(allManagers, activeManagers);
             }
+        } else if(galasaTest.isGherkin()) {
+            GherkinStatements.register(galasaTest, this, allManagers, activeManagers);
```

Comment:
doesn't the http manager have to examine the feature to determine if it is required or not?   then call youarerequired